### PR TITLE
Acceptance test external repo with helm chart

### PR DESCRIPTION
**Pull request by luizbafilho**
**What changed?**
- adding acceptance test to check external repo deployment with helm

**Why?**

**How did you test it?**

**Release notes**

**Documentation Changes**

### CODE REVIEW

**Review by josecordaz — APPROVED** [review]
LGTM

File: test/acceptance/test/add_tests.go [hunk 1]
```diff
@@ -261,6 +261,61 @@ var _ = Describe("Weave GitOps Add Tests", func() {
...
+			Expect(waitForResource("apps", appName, WEGO_DEFAULT_NAMESPACE, INSTALL_PODS_READY_TIMEOUT)).To(Succeed())
+			Expect(waitForResource("configmaps", "helloworld-configmap", WEGO_DEFAULT_NAMESPACE, INSTALL_PODS_READY_TIMEOUT)).To(Succeed())
+		})
+
+		By("There is no .wego folder in the app repo", func() {
+			_, err := os.Stat(repoAbsolutePath + "/.wego")
+			Expect(os.IsNotExist(err)).To(Equal(true))
+		})
+	})
+
```
Comment (iahmad9) [inline test/acceptance/test/add_tests.go:318]: How about checking that remote also doesn’t have .wego?

Comment (luizbafilho) [reply test/acceptance/test/add_tests.go:318]: it has to have that 😄

Comment (luizbafilho) [reply test/acceptance/test/add_tests.go:318]: But I'll check the existence of it, it's not really a `.wego` directory, it's a bit different.

Comment (iahmad9) [reply test/acceptance/test/add_tests.go:318]: Right, yeah somehow NONE was stuck in my mind 🙂. In that case presence of it.

**Pull request merged**